Simplenote/Classes/SimplenoteConstants.swift

```diff
@@ -41,4 +25,17 @@ class SimplenoteConstants: NSObject {
     /// Simplenote: Published Notes base URL
     ///
     static let simplenotePublishedBaseURL = "http://simp.ly/publish/"
+
+    /// AppEngine: Base URL
+    ///
+    static let currentEngineBaseURL: String = {
+        BuildConfiguration.current == .internal ? SPCredentials.experimentalEngineURL : SPCredentials.defaultEngineURL
+    }()
+
+    /// AppEngine: Endpoints
+    ///
+    static let resetPasswordURL     = currentEngineBaseURL + "/reset/?redirect=simplenote://launch&email="
+    static let settingsURL          = currentEngineBaseURL + "/settings"
+    static let signupURL            = currentEngineBaseURL + "/account/request-signup"
+    static let verificationURL      = currentEngineBaseURL + "/account/verify-email/"
```

Comment:
Nitpick.

In these instances, I find it useful to use [`NSString` `appendingPathComponent(_:)`](https://developer.apple.com/documentation/foundation/nsstring/1417069-appendingpathcomponent) instead of raw string manipulation because it handles the case of a trailing `/` in the base URL.

Reply:
Done, thank you!